test/unit/middlewares/articlesMiddleware.test.js

```diff
@@ -149,6 +151,27 @@ describe('API endpoint: /api/articles (Middleware test)', () => {
     });
   });
 
+  describe('POST: /api/v1/articles', () => {
+    describe('create an article without the category', () => {
```

Comment:
#### Nitpick
- Can we add tests for other use-cases for instance if the category doesn't exist.

Reply:
I am on it now